flytekit/bin/entrypoint.py

```diff
@@ -262,6 +263,31 @@ def _execute_task(task_module, task_name, inputs, output_prefix, raw_output_data
                 _handle_annotated_task(task_def, inputs, output_prefix, raw_output_data_prefix)
 
 
+@_scopes.system_entry_point
+def _execute_map_task(task_module, task_name, inputs, output_prefix, raw_output_data_prefix, max_concurrency, test):
+    with _TemporaryConfiguration(_internal_config.CONFIGURATION_PATH.get()):
+        with _utils.AutoDeletingTempDir("input_dir"):
```

Comment:
we do not need the temporary configuration or autodeleting tempdir for the new code right? can we limit this for the old code, that will make it easier to delete in the future

Reply:
oops copy/pasta. done.